packages/accounts-base/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "A user account system",
-  version: "1.6.0",
+  version: "1.7.0",
```

Comment:
I'm ok with 1.6.1 so we don't need a new Meteor version to use it.

Reply:
If it goes into Meteor 1.10.3 then I'm fine with that in order to keep up with semantic versioning.